src/pages/courses/courses.js

```diff
@@ -67,154 +74,939 @@ const useStyles = makeStyles(theme => ({
   },
   expandOpen: {
     transform: "rotate(180deg)"
+  },
+  toolbarItem: {
+    display: "flex",
+    justifyContent: "center",
+    [theme.breakpoints.down("xs")]: {
+      "&": {
+        justifyContent: "flex-start"
+      }
+    }
+  },
+  addIconFab: {
+    width: "45px",
+    height: "45px"
+  },
+  notFoundCourses: {
+    padding: "15px"
+  },
+  cardContent: {
+    display: "flex",
+    flexDirection: "column",
+    "& > *": {
+      padding: "5px 0 5px 0"
+    }
+  },
+  centerIconText: {
+    display: "flex",
+    alignItems: "center",
+    alignContent: "center"
+  },
+  date: {
+    "& > svg": {
+      marginRight: "10px"
+    },
+    "& > svg:nth-child(2)": {
+      marginLeft: "10px"
+    }
+  },
+  marginSvgIcon: {
+    "& > svg": {
+      marginRight: "10px"
+    }
+  },
+  deleteIcon: {
+    color: "#dc5a5a"
+  },
+  updateIcon: {
+    color: "#4CAF50"
+  },
+  removeCourseDisagree: {
+    color: "#dc5a5a"
+  },
+  removeCourseAgree: {
+    color: "#4CAF50"
+  },
+  boxCreateCourse: {
+    width: "70%",
+    [theme.breakpoints.down("xs")]: {
+      "&": {
+        width: "100%"
+      }
+    }
+  },
+  checkedBox: {
+    display: "flex",
+    flexDirection: "column"
+  },
+  checkedError: {
+    fontSize: "0.75rem",
+    marginTop: "3px",
+    fontFamily: "Roboto",
+    fontWeight: "400",
+    lineHeight: "1.66",
+    letterSpacing: "0.03333em",
+    color: theme.palette.errorLight
+  },
+  linearProgress: {
+    margin: "16px"
   }
 }));
```

Comment:
I suggest you split `styles` from the core component. This will save some lines, which is kinda important if we think of legibility of the code.